packages/rn-tester-e2e/tests/helpers/utils.js

```diff
@@ -38,10 +36,11 @@ class Utils {
 
   async scrollToElement(locator: string): Promise<void> {
     let {width, height} = await driver.getWindowSize();
+    console.log(width, height)
     let elementIsFound;
     try {
-      elementIsFound = await driver.$(locator).isDisplayed();
-      while (!elementIsFound) {
+      elementIsFound = await $(locator).isDisplayed();
+      while (!(await elementIsFound)) {
         driver.touchPerform([
           {
             action: 'press',
```

Comment:
I would suggest changing this scroll logic completely. Taking the window size and performing mathematical operations on it to determine touches is prone to error. Here's a better way:

        await browser.waitUntil(
          async () => {
            await driver.execute('mobile: scroll', { direction: 'down' });
            return await $(locator).isDisplayed();
          },
        );

Reply:
I've tried to change scroll logic with your or similar solutions, but after testing, the current implementation is less flak, at least in the local environment.